apps/web/src/components/(dashboard)/avatar/stack-avatar.tsx

```diff
@@ -23,12 +26,32 @@ export const StackAvatar = ({ first, zIndex, fallbackText, type }: StackAvatarPr
       break;
   }
 
+  switch (zIndex) {
```

Comment:
Let's just use an object outside of the component for this? so it can be `ZIndexes = { ... }; ZIndexes[key] ?? ''`